src/set_theory/ordinal_notation.lean

```diff
@@ -140,32 +140,33 @@ inductive NF_below : onote → ordinal.{0} → Prop
   ordinal notations, but to avoid complicating the algorithms
   we define everything over general ordinal notations and
   only prove correctness with normal form as an invariant. -/
-@[class, pp_nodot] def NF (o : onote) := Exists (NF_below o)
+class NF (o : onote) : Prop := (out : Exists (NF_below o))
+attribute [pp_nodot] NF
```

Comment:
Apparently `class` has some unnecessary restrictions regarding attribute use on the declaration, which is why this attribute is applied after the fact.